<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>poseidon</class>
 <widget class="QMainWindow" name="poseidon">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>poseidon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_choose_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择文件:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_choose_file">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>461</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_choose_file">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>750</width>
      <height>500</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_shezhi"/>
    <addaction name="action_tuichu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_tuichu">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_shezhi">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
